homeassistant/components/netgear_lte/sensor_types.py

```diff
@@ -10,17 +12,24 @@
     'rx_level': 'dBm',
     'tx_level': 'dBm',
     'upstream': None,
-    'wire_connected': None,
-    'mobile_connected': None,
     'connection_text': None,
     'connection_type': None,
     'current_ps_service_type': None,
     'register_network_display': None,
-    'roaming': None,
     'current_band': None,
     'cell_id': None,
 }
 
-ALL = list(SENSOR_UNITS)
+BINARY_SENSOR_MOBILE_CONNECTED = 'mobile_connected'
+
+BINARY_SENSOR_CLASSES = {
+    'roaming': None,
```

Comment:
I am in doubt about the proper device class to use here. The UI currently shows a checkmark when roaming and that seems odd. However, "roaming" is the terminology normally used so I see no way of reversing the naming without causing confusion.

Reply:
I don't have a better solution here.